menu.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
-
 <head>
 	<title>PizzaParty</title>
 	<link href="css/style.css" rel="stylesheet">
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 </head>
 <meta charset="UTF-8">
 <title>Menu</title>
```

Comment:
Is this supposed to be here? If so where is the opening body tag

Reply:
line 9 removed